regression/run-all.sh

```diff
@@ -27,10 +27,17 @@ find "$tstDir" -maxdepth 1 -mindepth 1 -type d ! -name ".*" -print0 \
             exit 0
         fi
         echo "[TEST]: $1"
+        OUTARGS=""
+        if [ ! -z "$GEN" ]; then
+            OUTARGS=("-o" "$1")
+        elif [ ! -z "$SAVE" ]; then
+            mkdir -p "$1/out"
+            OUTARGS=("-o" "$1/out")
+        fi
         if [ -f "$1"/scale ] ; then
-            "$2" "$1" -s "$(cat "$1"/scale)" -p "$ART_REG_TOLERANCE"
+            "$2" "$1" -s "$(cat "$1"/scale)" -p "$ART_REG_TOLERANCE" ${OUTARGS[*]}
         else
-            "$2" "$1" -p "$ART_REG_TOLERANCE"
+            "$2" "$1" -p "$ART_REG_TOLERANCE" ${OUTARGS[*]}
```

Comment:
This seems unrelated to the new tests. It also uses bash-specific arrays making it nonportable. *(By not utilising `"${OUTARGS[@]}"` expansion preserving members with spaces, it's actually no better than a plain string variable later expanded without quotes)*

Reply:
Whoooops, didn't mean to include that in this PR.